element_moseq/moseq_train.py

```diff
@@ -213,42 +214,54 @@ def make(self, key):
         kpms_root = moseq_infer.get_kpms_root_data_dir()
         kpms_processed = moseq_infer.get_kpms_processed_data_dir()
 
-        kpms_project_output_dir = (PCATask & key).fetch1("kpms_project_output_dir")
-        try:
-            kpms_project_output_dir = find_full_path(
-                kpms_processed, kpms_project_output_dir
-            )
-
-        except:
-            kpms_project_output_dir = kpms_processed / kpms_project_output_dir
+        kpms_project_output_dir, task_mode = (PCATask & key).fetch1(
+            "kpms_project_output_dir", "task_mode"
+        )
 
-        kpset_dir = find_full_path(kpms_root, kpset_dir)
-        videos_dir = find_full_path(kpms_root, Path(video_paths[0]).parent)
+        if task_mode == "trigger":
+            try:
+                kpms_project_output_dir = find_full_path(
+                    kpms_processed, kpms_project_output_dir
+                )
+
+            except:

```

Comment:
Typically, bare `except` blocks are not recommended. Is there a specific error you're trying to catch here?

Reply:
Yes, the code is designed to handle situations where the full path of `kpms_project_output_dir` cannot be found. If it encounters an error during the attempt to find the full path, it generates a path that doesn't currently exist, but it will be created later within the `setup_project` function (lines 234-237).

Reply:
would you catch that specific exception, `FileNotFoundError`?

Reply:
That's a good idea. Just like in [this example](https://github.com/datajoint/element-calcium-imaging/blob/3b420a7bd6a2e3b18f2485c395070deb9a15d72b/element_calcium_imaging/imaging.py#L317-L318), I've also included the exception in [this commit](https://github.com/datajoint/element-moseq/pull/4/commits/fd4f0c5e4aa5bac3a84148384e4ccef58c89aa18).